### DISCUSSION TITLE
"You appear not to be near any campus location for the school you have selected"....

### DISCUSSION THREAD

**Discussion by ghost** [community/community#68447] [GitHub Education]
### Hi everyone,

Hello, I just just try to appeal for GitHub student and I got this error: "You appear not to be near any campus location for the school you have selected"
I try on different browser and that change nothing, I live in France, Marseille and when I go on this website <https://www.gps-coordinates.net/my-location> that ask my location **LIKE** <https://education.github.com> the website return France, Paris (~434.96 Miles or 700 KM away from where I live) 
can I get some help ? Thanks you !

and no I'm not using a VPN or something like that

**Comment by mantasu** [comment]
Hi, have you got your location enabled (in both settings and through the browser)? Maybe you could try registering using your mobile phone?

If you still experience issues, even though your location is correct, I would suggest opening a [ticket](https://support.github.com/request/education?tags=docs-generic).

**Reply by ghost** [reply]
Hey I just try and same... I try on multiple browser and multiple phone and same error (+ When I try to use the back camera, it just didn't work)

**Reply by mantasu** [reply]
Try checking on a different WiFi/mobile data. If you entered any school/university details, double check it as some universities use similar names (e.g., it could be College). Finally, try registering when you actually are on campus, e.g., in your school library. At least that's what I would try further.
